hack/generate.sh

```diff
@@ -47,3 +47,7 @@ protoc --go_out=plugins=grpc:. pkg/handler-launcher-com/cmd/info/info.proto
 
 mockgen -source pkg/handler-launcher-com/notify/info/info.pb.go -package=info -destination=pkg/handler-launcher-com/notify/info/generated_mock_info.go
 mockgen -source pkg/handler-launcher-com/cmd/info/info.pb.go -package=info -destination=pkg/handler-launcher-com/cmd/info/generated_mock_info.go
+
+rm -rf cluster-up
+#TODO move this to official kubevirt/kubevirtci repo
+curl -L https://github.com/davidvossel/kubevirtci/archive/${kubevirtci_git_hash}/kubevirtci.tar.gz | tar xz kubevirtci-${kubevirtci_git_hash}/cluster-up --strip-component 1
```

Comment:
@rmohr here's the logic we discussed. I've moved all the configs and dynamic data generated by cluster-up into a separate directory entirely (defaults to _ci-configs). THis means nothing is lost by deleting cluster-up/ and re-creating it during 'make generate'

for more info on the config's dir change, see this kubevirtci PR, https://github.com/kubevirt/kubevirtci/pull/88